packages/flame/lib/src/components/component.dart

```diff
@@ -133,6 +157,8 @@ class Component {
     removeFromParent();
   }
 
+  //#region Debug rendering
```

Comment:
What IDE are you using @st-pasha ?

Reply:
Android Studio -- I believe Erick taught me this trick some long time ago. Does this annotation not work in any other IDE?